src/util/cover-model.js

```diff
@@ -120,3 +120,45 @@ export default class CoverEntity {
     this.hass.callService("cover", service, data);
   }
 }
+
+export function supportsOpen(stateObj) {
```

Comment:
- let's create a helper to check support for a feature
- Prefer to use an arrow function, as they are lighter to execute.

```js
const support = stateObj, feature => (stateObj.attributes.supported_features & feature) !== 0;

export const supportsOpen = stateObj => support(stateObj, 1);
```


 `export const supportsOpen = stateObj =>